### DISCUSSION TITLE
Student re-verification form incorrect. No response from support for 3 weeks

### DISCUSSION THREAD

**Discussion by mikellykels** [community/community#61074] [GitHub Education]
I've submitted a ticket regarding my student re-verification that the form is incorrect when I select my school. I have not received a response in over 3 weeks and I have asked for updates each week yet nothing. A ticket was even "escalated" to the education department, but still no response. Is there even a support team in that department?

The form says I need a school domain however, the school does not give students email addresses, therefore I need to use my personal email. I used it previously for the same school, which worked at that time.

I'm also not provided an option on the form to upload any documents to provide proof of enrollment.

My student status is supposed to end at the end of this month. I thought this would have been resolved almost a month ago!

**Comment by nonvenomous** [comment]
Same issue here.
There are many people with this issue, basically every 4th discussion in the github education category is about this.
A big thread here https://github.com/orgs/community/discussions/59360

Where did you submit a ticket @mikellykels? I would like to do that too.

**Reply by mikellykels** [reply]
@nonvenomous Thanks for that thread - commented on it to follow the discussion. As for submitting the ticket, I submitted it after selecting the "Contact us" option https://support.github.com/request/education?tags=hubberfy_education. However, I haven't gotten a response there. You might get better luck trying to get _someone_ to respond from their general support https://support.github.com/request, but they might just kick it to the education department where no one responds again...

**Reply by nonvenomous** [reply]
I will give it a try. Thank you!

**Comment by samus-aran** [comment]
I'll be closing this discussion due to lack of recent activity. We can confirm that you have since been successful - enjoy your journey through GitHub Education.
